src/main/scala/firrtl/Emitter.scala

```diff
@@ -111,1446 +97,33 @@ object EmitAllModulesAnnotation extends HasShellOptions {
 }
 
 // ***** Annotations for results of emission *****
-sealed abstract class EmittedComponent {
-  def name:         String
-  def value:        String
+abstract class EmittedComponent {
```

Comment:
Please keep this sealed, it will involved moving `EmittedFirrtlCircuit` back to this file